modules/ROOT/pages/start/gs-sgw-api-access.adoc

```diff
@@ -12,13 +12,13 @@ include::partial$block-abstract.adoc[]
 
 :xref-pfx: {xref-pfx-sgw}:
 
-Sync Gateway APIs are accessed on different TCP ports. This makes it easy to expose the Public REST API (port 4984) to endpoints while keeping the Admin REST API (port 4985) secure behind your firewall.
-
-include::partial$block-prometheus.adoc[]
+Sync Gateway REST APIs are accessed on different TCP ports.
+This makes it easy to expose the Public REST API (port 4984) and Metrics REST API (port 4986) to endpoints while keeping the Admin REST API (port 4985) secure behind your firewall.
```

Comment:
This reads to me like the metrics endpoint is available on 4986 but you have the option to change it. Should make clear that this is not exposed unless the option is specified.

Reply:
Amended